Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.mm

```diff
@@ -136,13 +140,77 @@ @interface UIViewController ()
     videoDimensionsChanged(model ? model->videoDimensions() : FloatSize());
 }
 
+void VideoPresentationInterfaceIOS::ensurePipPlacardIsShowing()
+{
+    if (m_pipPlacard) {
+        [m_pipPlacard setHidden:NO];
+        return;
+    }
+    auto pipPlacard = adoptNS([PAL::allocUIViewInstance() initWithFrame:[layerHostView() bounds]]);
```

Comment:
Prefer explicit `RetainPtr` over `auto`, due to smart pointer guidelines. Couple instances throughout the patch.